src/outputs/outputs_main.py

```diff
@@ -82,6 +82,37 @@ def run_outputs(
         )
         OutputMainLogger.info("Finished status filtered output.")
 
+    # Topping up the postcodes
+    def _topup(mystr: str, target_len: int = 8):
+        ''' Regulates the number of spaces in a postcode, so that the total
+        length is 8 characters.
+        Brings all letters to upper case, in line with the mapper.
+        Splits it into parts, separated by any number of spaces.
+        If there are two or more parts, the first two are used.
+            If their total lenght is too big, cuts the tail
+        If it's only one part, keeps 8 characters, tops up with spaces if needed
+        '''
```

Comment:
Docstring needs reformatting into the standard format.

Reply:
Updated the docstring